test/models/monitoring/AppealAuditModelSpec.scala

```diff
@@ -155,33 +156,60 @@ class AppealAuditModelSpec extends SpecBase {
       ))
     )
 
+    val seqOfUploads: Seq[UploadJourney] = Seq(
+      UploadJourney(
+        reference = "ref1",
+        fileStatus = UploadStatusEnum.READY,
```

Comment:
the audit json object should be generated for UploadJourney in case of READY or DUPLICATE but it seems we also get values for status WAITING or FAILED 
 "noOfUploadedFiles": "2",           
 "uploadedFiles": [Some values for two files uploaded]

Reply:
Filtering of status is done in AuditService so no need to hanle here, have added extra Test case on the AuditService to cover this check